lib/recurly/validate.js

```diff
@@ -15,8 +15,25 @@ const debug = require('debug')('recurly:validate');
 const TYPES = [
   {
     type: 'discover',
-    pattern: /^(3[0689]|6011|621094|62[24568]|64[4-9]|65|81[0-7])/,
-    lengths: [14, 15, 16, 17, 18, 19]
+    pattern: {
+      test: (n) => {
+        let firstEight = parseInt(n.substr(0, 8), 10);
+        let accepted_ranges = [
+          [60110000,60110399],
+          [60110500,60110999],
+          [60112000,60114999],
+          [60117400,60117499],
+          [60117700,60117999],
+          [60118600,60119999],
+          [64400000,65059999],
+          [65060100,65060999],
+          [65061100,65999999]
+        ];
+
+        return accepted_ranges.find(([minimum, maximum]) => firstEight >= minimum && firstEight <= maximum) !== undefined
```

Comment:
I think there needs to be a semi-colon at the end of this line to pass the lint step in our build https://travis-ci.org/recurly/recurly-js/jobs/654252599?utm_medium=notification&utm_source=github_status